docs/proposals/custom-wbt-settings.md

```diff
@@ -0,0 +1,50 @@
+# Proposal: Support option to change WBT settings
+
+Last updated: 07.12.2021
+
+
+## Abstract
+
+CSI should have an opportunity to automate WBT (writeback throttling) disabling for specific Volumes.
```

Comment:
For specific SCs I guess and not disabling, but managing.

Reply:
fixed